README.md

```diff
@@ -1,41 +1,41 @@
-# Resteasy
+# RESTEasy
 
 [![Build Status](https://travis-ci.org/resteasy/Resteasy.svg?branch=master)](https://travis-ci.org/resteasy/Resteasy)
 
-Resteasy is a JBoss.org project aimed at providing productivity frameworks for developing client and server RESTful applications and services in Java.  It is mainly a JAX-RS implementation but you'll find some other experimental code in the repository.
+RESTEasy is a JBoss.org project aimed at providing productivity frameworks for developing client and server RESTful applications and services in Java.  It is mainly a JAX-RS implementation but you'll find some other experimental code in the repository.
 
 The project page can be found at http://resteasy.jboss.org
 
 ## JAX-RS
-RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a new JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
+RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
 
 Read the entire specification [here](https://jax-rs-spec.java.net/)
 
-The JAX-RS code is in the jaxrs directory.
+The RESTEasy, JAX-RS code is in the github repository, (https://github.com/resteasy/Resteasy.git).
```

Comment:
I propose: "The RESTEasy code is in the github repository, (https://github.com/resteasy/Resteasy.git)"